### PR TITLE
Changed docker image

### PR DESCRIPTION
**Pull request by npanpaliya**

### CODE REVIEW

File: conda-recipes/tensorflow-datasets-feedstock/ci_support/run_docker_build.sh [hunk 1]
```diff
@@ -50,7 +50,7 @@ docker run ${DOCKER_RUN_ARGS} \
                         -e HOST_USER_ID \
                         -e UPLOAD_PACKAGES \
                         -e CI \
-                        -a stdin -a stdout -a stderr \
+                        -a stdin -a stdout -a stderr  --privileged=true -u root \
```
Comment (wdirons) [inline conda-recipes/tensorflow-datasets-feedstock/ci_support/run_docker_build.sh]: Interesting how the travis failure mentions tty. Can you try undoing the TTY change we made last time for the anvil container?

Comment (npanpaliya) [reply conda-recipes/tensorflow-datasets-feedstock/ci_support/run_docker_build.sh]: I tried that too and the old error appears "input device is not tty"

Comment (npanpaliya) [reply conda-recipes/tensorflow-datasets-feedstock/ci_support/run_docker_build.sh]: Removing --privileged=true worked.

File: conda-recipes/tensorflow-datasets-feedstock/ci_support/build_steps.sh [hunk 2]
```diff
@@ -20,6 +20,7 @@ export FEEDSTOCK_ROOT=/home/conda/feedstock_root
 export RECIPE_ROOT=/home/conda/recipe_root
 export CI_SUPPORT=/home/conda/feedstock_root/.ci_support
 export CONFIG_FILE="${CI_SUPPORT}/${CONFIG}.yaml"
+export PATH=/opt/anaconda/bin:$PATH
```
Comment (deepali-c) [inline conda-recipes/tensorflow-datasets-feedstock/ci_support/build_steps.sh]: Please update copyright.

**Pull request merged**